gluon/main.py

```diff
@@ -390,7 +390,11 @@ def wsgibase(environ, responder):
                                    % 'invalid request',
                                    web2py_error='invalid application')
                 elif not request.is_local and exists(disabled):
-                    raise HTTP(503, "<html><body><h1>Temporarily down for maintenance</h1></body></html>")
+                    if os.path.exists(os.path.join(request.folder,'static','503.html')):
+                        import urllib
+                        raise HTTP(503, urllib.urlopen(os.path.join(request.folder,'static','503.html')).read())
```

Comment:
(I never saw these until just now...)

@niphlod You actually have a really good point...that's what I get for 4:30 AM code-writing.  XD